### ISSUE TITLE
Can't install air file with runtime 33.1.1.444 under Windows 10

### ISSUE THREAD

**Issue by ozster123** [airsdk/Adobe-Runtime-Support#833]
## Problem Description

Describe your problem in detail. Include the following information:

* Which AIR SDK did you use? 33.1.1.444 (Runtime)
* Describe the affected environment: AIR Windows 10
* Can you reproduce it yourself, or did you get reports from other users? Several customers reported the issue. I also reproduced the issue myself.

## Steps to Reproduce

Install AIR Runtime 33.1.1.444.
Try to install an .air file. User gets the Application cannot be installed due to damaged installer file. Same air file and runtime version works on the Mac. Also I tried installing the air file before upgrading to 33.1.1.444 and it works.

![image](https://user-images.githubusercontent.com/10943652/117140902-d46c2b80-ada5-11eb-9634-43eaaaac5a44.png)

## Known Workarounds

I would like to try to uninstall 33.1.1.444 and reinstall 33.1.1.385 but I can't find a link to this runtime. Any ideas where I can find it?

**Comment by ajwfrost** [comment]
Hi
Sorry, yes the .444 shared runtime had an error in the product IDs and DLL version numbering which causes a few problems. For resolution and to get one of the other installers, please see the below link
https://transfer.harman.com/message/BYboRa76qHPhoGVLJ9CpSN
thanks
  Andrew

**Comment by TEAM-OUTDOOR** [comment]
Hello,

Thanks for the link of the new runtime.
I have uninstall the 33.1.1.44 and then install the 33.1.1.476.
Now in the control panel, it is the good version appearing, and it is possible to uninstall it from the panel directly.

But I still have the problem that I get the message from @ozster123 when trying to install an AIR package (for example https://downloads.harksolutions.net/aWorkbook_WIN_8_6_1.air).

Do you have an idea why the package cannot be installed with 33.1.1.476 and is installing well with version 32.0.0.125?

Regards.

**Comment by ajwfrost** [comment]
Curious, it worked for the test packages that we've created, but with that .air file it's showing in the logs:
Migration signature validation failed

Will take a look: we still have an issue with the certificates and XML signature validation that Adobe had in place which isn't working for us, partly due to an expired certificate and partly due to source code licensing (which we may have a workaround for).

**Comment by ajwfrost** [comment]
Yes it depends on how the signing was done, apparently. We've got past the failure above but for some reason these installs aren't then triggering the evaluation to an Admin user, they're just aborting.. (works in debug builds, naturally, but fails in release builds...). Once that's fixed, we'll get an updated shared runtime install created and uploaded!

thanks

**Comment by ajwfrost** [comment]
Updated runtime that fixes these issues - we should be able to upload this to our website early next week but in the meantime, please check at:
https://transfer.harman.com/message/HsoPhfezkGvFARx78GpRDa
thanks

**Comment by ozster123** [comment]
That version works for me. Cheers.

**Comment by TEAM-OUTDOOR** [comment]
Hello.
I can confirm that it is also OK for me.
Thank you for the correction.
Regards.

**Comment by stevenspeirs** [comment]
Hi.
Unfortunately, I'm still having trouble. I've tried the 33.1.1.476 and 33.1.1.497 runtimes (on Windows 10) and I am still unable to install JuiceboxBuilder.air (inside the zip file here: https://juicebox.net/release/juicebox_lite_1.5.1.zip) or any of the sample AIR apps that I downloaded from Adobe's site before they took it down.
33.1.1.476 and 33.1.1.497 both install and uninstall OK (unlike 33.1.1.444) but I just get the 'installer file is damaged' message when trying to install the apps I have tried.
Reverting to 33.1.1.385 works fine.
I hope you can help. Thanks!

**Comment by ajwfrost** [comment]
Thanks @stevenspeirs  - odd as this one is again showing:
```
[2021-05-16:07:04:14] Package signature validation failed
[2021-05-16:07:04:14] Got an unexpected fatal error while unpackaging: [ErrorEvent type="error" bubbles=false cancelable=false eventPhase=2 text="invalid package signature" errorID=5022]
```
and I thought this had already been handled.. so perhaps there's another check that's throwing this error.

**Comment by ajwfrost** [comment]
@stevenspeirs  please try with the below one! I believe this one covers all the recent updates and will allow all .air packages to be displayed/installed..
https://transfer.harman.com/message/yAjiByOKfZMNvio19j765S

thanks

**Comment by stevenspeirs** [comment]
@ajwfrost Thank you so much for the quick turnaround on this.
I have tried v33.1.1.502 and it works well. I can now install JuiceboxBuilder-Lite and the other sample AIR apps that I have.
Just for good measure, I tried v33.1.1.502 as a clean install and as an upgrade to v33.1.1.444. It worked well in both scenarios.
Thanks, once again.

**Comment by stevenspeirs** [comment]
@ajwfrost
Hi.
The links/messages you have previously posted in this thread (to v33.1.1.385 and v33.1.1.502) have all expired.
Is there any chance you could please re-upload a link to v33.1.1.385 (or v33.1.1.502) so that I can point our Windows users towards a working AIR runtime until a new one is published on your regular download page?
Many thanks!

**Comment by ajwfrost** [comment]
Hi
Our website lead engineer is still off, his return to work keeps being delayed, so we finally got someone else to go through and work out the updates.. so the 33.1.1.502 installer is up there now.
Direct link: https://airsdk.harman.com/assets/downloads/AdobeAIR.exe
thanks

**Comment by stevenspeirs** [comment]
That's great! Thank you.

**Comment by ajwfrost** [comment]
Should be fixed in 33.1.1.533, please check and re-open if necessary.
Thanks

**Comment by TCreutzenberg** [comment]
I already wrote this in another issue but I will just copy it here so people save hours of trying to fix the same thing:

The problem is the "Windows XX N" [N]-version. This is an optional Windows version for the European market without the media pack. After installing the Windows Media Pack (optional feature under Windows "N") the installation and the AIR runtime including WebKit (version 33.1.1.744) works just fine.
I hope this helps somebody running into the same problem!
